app/features/history/history_filter/tags/tagsToInclude/EditHistoryFilterTagsToIncludeScreen.js

```diff
@@ -0,0 +1,39 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import SelectTagsModal from '../SelectTagsModal';
+import * as SetsSelectors from 'app/redux/selectors/SetsSelectors';
+import * as Actions from './EditHistoryFilterTagsToIncludeActions';
+import * as HistorySelectors from 'app/redux/selectors/HistorySelectors';
+import * as OneRMCalculator from 'app/math/OneRMCalculator';
+
+const mapStateToProps = (state) => {
+    const exercise = HistorySelectors.getEditingExerciseName(state);
+    
+    // TODO move tags suggestion out
+    return {
+        title: 'Tags to Include',
+        placeholder: 'Enter Tag',
+        text: '',
+        inputs: HistorySelectors.getEditingFilterTagsToInclude(state),
+        generateSuggestions: (input, ignore) => OneRMCalculator.getTagsToIncludeSuggestions(state, exercise, input, ignore),
```

Comment:
- [ ] Wait I don't think this should be using 1RM tags to include

Reply:
Did this for quickness, it's use would be identical gonna have to refactor it

Reply:
Going to do this in a new pr since I have to build a new suggestions generator for History Filter